knowledge-base/pass-sqldatasource-columns-as-datasource-for-nested-data-items.md

```diff
@@ -0,0 +1,61 @@
+---
+title: How to Use Part of the Columns from a Parent SqlDataSource as DataSource for Nested Data Items
+description: When a parent data item is bound to a SqlDataSource you may still use part of the returned columns as DataSource for child data items
+type: how-to
+page_title: How to Pass SqlDataSource Columns as DataSource for Nested Data Items
+slug: pass-sqldatasource-columns-as-datasource-for-nested-data-items
+position: 
+tags: 
+ticketid: 1454415
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+When you need to pass data from a [SqlDataSource](../sqldatasource) to its nested [data item](../data-items), you need to pass 
```

Comment:
When you need to pass data from an SqlDataSource bound to a data item to its nested data item